pkgs/misc/emulators/citra/base.nix

```diff
@@ -0,0 +1,73 @@
+{ pname, version, src, branchDesc, branchName
```

Comment:
branchName is not used anywhere.

Reply:
Thanks for the suggestion, I just remembered why this is here. 

The build system uses git to fetch the version number, there was an attempt to use a workaround like what was used for Yuzu to manually define the version number but I was having trouble with it and moved on for a time, see: https://github.com/NixOS/nixpkgs/blob/8a70a6808c884282161bd77706927caeac0c11e8/pkgs/misc/emulators/yuzu/base.nix#L58

I'd still like to get this working. I'll have another crack at it.

Reply:
You don't need to add this. Just drop the unused inputs for now so that we can get this merged.